### PR TITLE
remove ALL filter

### LINKED ISSUE #412: Remove `All` toggle
## Kind (Required)

- Improvement

 ## Overview (Required)

- Because it is different from the behavior of the check box.
All toggles are not needed. It exists in Figma, but the designer just can't erase it because she doesn't have time. 🙏 
image

 ## Links	

-

### PR DESCRIPTION
**Pull request by ked4ma**
## Issue
- close #412 

## Overview (Required)
- checkbox put on filter is not fit as checkbox behavior.
- so remove ALL filter from filter parts.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt [hunk 1]
```diff
@@ -70,24 +64,14 @@ private fun Input(
                     Color.Transparent
                 }
             )
-            .padding(all = 12.dp)
+            .padding(horizontal = 20.dp, vertical = 12.dp)
```
Comment (ked4ma) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:67]: when remove spacers (L83, L90), it seems to go left too much.
so i add additional padding here.

**Comment by takahirom** [comment]
I'm sorry. 🙇 
Removing the "All" Filter means designing it as follows. 
image

**Comment by ked4ma** [comment]
oh, i understood the spec of this issue.
i'll modify it.

**Comment by ked4ma** [comment]
@takahirom 
i removed ONLY all filter, and restored about favorite filter!

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt [hunk 2]
```diff
@@ -63,13 +55,6 @@ private fun Input(
         Modifier
             .clickable(onClick = onClick)
             .fillMaxWidth()
-            .background(
-                if (isChecked) {
-                    ConferenceAppFeederTheme.filterMuskColor
-                } else {
-                    Color.Transparent
-                }
-            )
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:72]: Thank you for applying the changes!
Is there any reason to remove the background? 👀

Comment (ked4ma) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:72]: i thought this background musk is used to show which filter is applied.
and by this change, there are only ONE filter item and by checkbox, we can recognize whether favorite filter is selected or not.
so, i changed as above.
is it better to rollback about backgound ?

Comment (takahirom) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:72]: I take a look at the material design guideline for a moment. :eyes:

Comment (takahirom) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:72]: Certainly, it's not in the guidelines either. Let's erase it once. :+1:

**Review by takahirom — APPROVED** [review]
LGTM!

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt (hunk 2 above)
Comment (ked4ma) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt:72]: thank you for your check!!!

**Pull request merged**